task1.js

```diff
@@ -0,0 +1,73 @@
+//  Write a JavaScript program to pick a specified number of unique random elements from a given array (so that no element is selected more than once).
+//  Sample Output:
+//  Original array:
+//  [12, 34, 23, 56]
+//  2 random elements from the array.
+//  [34, 12]
+//  3 random elements from the array.
+//  [56, 12, 34]
+
+//  If an element is already selected, it cannot be selected again.
+
+const findRandomElements = (arr, elemCount) => {
+  if (!Array.isArray(arr)) {
+    throw new Error("Array expected.");
+  }
+
+  if (!arr.length) {
+    throw new Error("Array is empty.");
+  }
+
+  if (elemCount > arr.length) {
+    throw new Error(
+      "Provided number of requested elements is bigger, than array length."
+    );
+  }
+
+  if (!Number.isInteger(elemCount) || elemCount <= 0) {
+    throw new Error(
+      "Provided 'elemCount' value must be positive integer, grater or equal to 0."
+    );
+  }
+
+  const uniqueElements = [...new Set(arr)];
```

Comment:
u don't need to use `Set `here
usique element - element is selected more than once
if you put a lot of  same elements,  it is ok